juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
+        rule aft_server_entry_stats {
+            synopsis "Aft Server Sandbox Entry Stats";
+            description "Aft Server Sandbox Entry Stats";
+            sensor aft_sb_entry_stats {
+                iAgent {
+                    file aft-sandbox-stats.yml;
+                    table SandboxStatsTable;
+                    frequency 180s;
+                }
+            }
+            field aft-entry {
```

Comment:
Field description is mandatory, please include description for all fields

Reply:
Done